extensions/wikia/AdEngine/js/lookup/openXBidder.js

```diff
@@ -157,7 +153,6 @@ define('ext.wikia.adEngine.lookup.openXBidder', [
 	return factory.create({
 		logGroup: logGroup,
 		name: 'ox_bidder',
-		isEnabled: isEnabled,
```

Comment:
It seems none of our bidders implements this method any longer. What do you think about [removing it from the factory as well](https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/lookup/lookupFactory.js#L23)?